docs/usage/docker.md

```diff
@@ -187,6 +187,25 @@ Add `"docker:enableMajor"` to your `extends` array.
 
 Add `"default:pinDigestsDisabled"` to your `extends` array.
 
+<!-- markdownlint-disable MD046 -->
+<!-- prettier-ignore -->
+!!! note
+    This preset only sets the global default for the [digest pinning flag](./configuration-options.md#pindigests) to `false`.
```

Comment:
I'm not sure if these relative links will work from here. Have you tested this via https://github.com/renovatebot/renovatebot.github.io/blob/21a665cbb0a8ec76b838302e0ac4e51ad75c6d8d/Makefile#L41